src/EventEdition/GuestGrid.vala

```diff
@@ -105,12 +107,16 @@ public class Maya.View.EventEdition.GuestGrid : Gtk.Grid {
                     if (address.value == mail_address) {
                         individual = map_iterator.get_value ();
                         if (individual != null) {
-                            try {
-                                individual.avatar.load (32, null);
-                                avatar = new Granite.Widgets.Avatar.from_file (individual.avatar.to_string (), 32);
-                            } catch (Error e) {
-                                critical (e.message);
-                            }
+                            avatar = new Hdy.Avatar (ICON_SIZE, individual.display_name, true);
```

Comment:
Since we've already created an avatar, we probably only need to update the text property:
```suggestion
                            avatar.text = individual.display_name;
```

Reply:
I've applied the suggested change.